stacks/spica/src/activity/interface.ts

```diff
@@ -0,0 +1,44 @@
+export interface Activity {
```

Comment:
I assume that you are going to change these interfaces within the upcoming changes. Right ?